project/AppDependencies.scala

```diff
@@ -3,13 +3,17 @@ import sbt._
 
 object AppDependencies {
 
+  private val catsVersion = "2.1.1"
+
   val compile = Seq(
-    "uk.gov.hmrc"       %% "bootstrap-play-26"   % "1.13.0"
```

Comment:
Why was that lined removed & restored?

Reply:
Because I added a comma to add the cats-core library